src/components/Transactions/SplitTransactionForm/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+
+import MultiSelectComponent from './multiSelect'
+import AsyncSelect from 'react-select/async'
+
+import API from '../../../API'
+import { FRIEND_LIST_URL } from '../../../Config'
+import { getRandomColor } from '../../Charts&Tables/Utils/chartUtils'
+
+class SplitTransactionForm extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: '',
+            total_amount: 0,
+            creator: localStorage.getItem('userid'),
+            category: 1,
+            paying_friend: null,
+            all_friends_involved: []
+        }
+    }
+
+    promiseFriends = (inputValue) => new Promise((resolve, reject) => {
+        const url = FRIEND_LIST_URL + `?search=${inputValue}`
+        API.fetchFriendsList(url).then(friendResult => {
+
+            const friends = friendResult.results.map((friend) => {
+                return { value: friend.id, label: friend.username, color: getRandomColor() }
+            })
+
+            resolve(friends.concat([{ value: localStorage.getItem('userid'), label: localStorage.getItem('username'), color: getRandomColor() }]))
+        })
+    })
+
+    handleChange = (name, value) => {
+        this.setState({
+            [name]: value
+        })
+    }
+
+    handleSubmit = async (event) => {
+        event.preventDefault();
+        const res = await API.createSplitTransaction(this.state)
+        if (res.status === 201) {
+            alert("Expense Splitted");
+            event.target.parentNode.parentNode.reset();
+            window.location.reload()
+        } else {
+            const error = await res.json()
+            alert(error[Object.keys(error)[0]])
+        }
+    }
+
+    render() {
+        return (
+            <form>
+                <div className='mb-3'>
+                    <label htmlFor='title'>Title</label>
+                    <input
+                        type='text'
+                        name='title'
+                        className='form-control'
+                        placeholder='Add title here'
+                        onChange={(event) => { this.handleChange(event.target.name, event.target.value) }}
```

Comment:
why are we sending value as `event.target.value` and not reading from state instead